### ISSUE TITLE
Cannot open big images, size limitet of LongInt, not Int64

### ISSUE THREAD

**Issue by jonyrh-td-sever** [galfar/imaginglib#38]
Cannot open big images, size limitet of LongInt, not Int64..

May bi, add compile defines for CPU x86_64 to use big images? 
And open images in stream as TFileStream for not use RAM?

**Comment by galfar** [comment]
That's right, size in `TImageData` should be `Int64` or `NativeInt` rather.
More troubles would be with some IO functions and some image format loaders/savers.
What are you most interested in?

I used memory mapped files as `Bits` in `TImageData` for big images in a project using Imaging many years ago - I'll see if it could be resurrected (at least as a demo).

**Comment by jonyrh-td-sever** [comment]
I think NativeInt will be more compatible, since you won't have to create separate code for different compiler directives like `{$DEFINE CPUX86_64}`. And thanks for the answer! :-)

**Comment by galfar** [comment]
Basic types and functions are ready for huge images now, with 6324e4e998997fecc3ce681a97325fb7e386c68a we can read  >4 GB BigTIFF files. 
Still a lot to go...
